README.md

```diff
@@ -117,6 +117,21 @@ result.paginate
 # => #<OpenStruct current_page=2, per_page=30, total_pages=2, total_count=60>
 ```
 
+Create a LessonProgress:
+```ruby
+data = {
+  enrollment_id: 2119734,
+  lesson_progress: {
+    lesson_id: 1161016,
+    completed: true,
+    progress: 100.0
+  }
+}
+
+Edools::LessonProgress.create(data)
```

Comment:
Rodei exatamente essa query e resultou em um sucesso, mas ao rodar novamente obtive um erro: `NoMethodError (undefined method `objectify_lesson_progress' for #<Edools::LessonProgress:0x0000559529bc2eb0>)`
Que seria quando  o servidor tá retornando 422. {"errors":{"enrollment_id":["já está em uso"]}}

Reply:
ajeitei aqui, acabei de mandar o commit e aproveitei para gravar um VCR para essa caso...